src/mmw/js/src/modeling/controllers.js

```diff
@@ -114,15 +114,37 @@ var ModelingController = {
             project = App.currentProject;
             project.set('model_package', modelPackage);
             itsiResetProject(project);
+            App.getMapView().addSidebarToggleControl();
```

Comment:
Good catch, I think that's redundant; I tried to totally separate ITSI/non-ITSI blocks once the project state calisthenics started getting awkward, this is a leftover from that.